src/CoreBundle/private/library/classes/MenuItems/TCMSMenuItem_Module.class.php

```diff
@@ -29,9 +29,6 @@ public function GetLink()
         if ('' !== $pagedefType) {
             $url .= '&_pagedefType='.$pagedefType;
         }
-        if ('1' == $this->data['show_as_popup']) {
```

Comment:
This has been doing what?

Reply:
Did open modules in an overlay. Now all modules are opened as full page, with the default browser option to open in a new tab while holding CTRL key.